city_scrapers/spiders/det_employment_solutions.py

```diff
@@ -0,0 +1,184 @@
+import re
+from datetime import datetime
+
+from city_scrapers_core.constants import ADVISORY_COMMITTEE, BOARD, COMMITTEE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class DetEmploymentSolutionsSpider(CityScrapersSpider):
+    name = "det_employment_solutions"
+    agency = "Detroit Employment Solutions Corporation"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.descmiworks.com/about-us/public-meetings/"]
+    curr_type = None
+
+    MEETING_TYPES = ["MWDB", "CEAC", "DESC"]
+    MEETING_TYPE_TO_CSS_SELECTOR_MAP = {
+        "MWDB": "div.mwdb-meeting_wrapper_inner-wrapper",
+        "CEAC": "div.elementor-text-editor",
+        "DESC": "div.elementor-text-editor > p:nth-child(even) > a",
+    }
+
+    def get_meeting_type(self, item):
+        """Get the meeting type based on the item selector."""
+        if not item:
+            return None
+        info = item.get()
+        # There are 3 meeting types mentioned on the webpage
+        if "mwdb" in info:
+            return self.MEETING_TYPES[0]
+        if "CEAC" in info:
+            return self.MEETING_TYPES[1]
+        if "DESC" in info:
+            # DESC meetings currently have parsable links only
+            return self.MEETING_TYPES[2]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        NOTE: If the HTML structure changes this will need to be re-written.
+
+        NOTE: The entire monthly schedules are ONLY available for download
+        in a docx and pdf format.
+
+        Parsable meeting types:
+         - MWDB Meetings
+         - CAEC Meetings
+
+        Unparsable meetings types (e.g. PDF/DOCX):
+         - DESC Meetings
+        """
+        elementor_row = response.css(
+            "section.elementor-element:nth-child(4)"
+            + "> div:nth-child(1) > div:nth-child(1)"
+        )
+        # The css selector for the left section of the page
+        meeting_sections = elementor_row.css("section.desc-public-meeting")
+        # Assumes location is the same across meeting types.
+        location = self._parse_location(meeting_sections[0])
+        # Appends the css selector for the right section on the page
+        meeting_sections.append(
+            elementor_row.css("div.elementor-row > div.elementor-element-77d4b08")
```

Comment:
It looks like the `77d4b08` portion of this is auto-added by a CMS, which makes me think it could change if edited, but it looks like pulling the first `section` tag in the second element with `[data-element_type="column"]` could be more stable? Since we're iterating through you could also just pull all sections in that group

Reply:
Your suggestion is definitely more stable. Good point about the auto-added CMS thing!